pkg/apps/store/store.go

```diff
@@ -1,7 +1,13 @@
 package store
 
 import (
+	"errors"
 	"fmt"
+	v12 "github.com/blackducksoftware/synopsys-operator/pkg/api/blackduck/v1"
+	v13 "github.com/blackducksoftware/synopsys-operator/pkg/api/size/v1"
+	"github.com/blackducksoftware/synopsys-operator/pkg/size"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
```

Comment:
Can we rename meta/v1 v1 to metav1, blackduck/v1 v12 to blackduckapi and size/v1 v13 to sizeapi? v12, v13 are confusing

Reply:
done